pkg/plugins/policies/meshmetric/plugin/v1alpha1/testdata/otel_and_prometheus.listeners.golden.yaml

```diff
@@ -75,15 +74,15 @@ resources:
             virtualHosts:
             - domains:
               - '*'
-              name: system_meshmetric_otel_otel-collector-observability-svc-4317
+              name: _kuma:metrics:opentelemetry:otel-collector.observability.svc-4317
```

Comment:
Why are these changing suddenly?

Reply:
I've changed old test to not use unified naming and added specific case for unified naming